src/data_management/rasters/RasterLayerTable.tsx

```diff
@@ -165,6 +206,57 @@ export const RasterLayerTable = (props:any) =>  {
           ]}
           newItemOnClick={handleNewRasterClick}
         />
+        { 
+        rowsToBeDeleted.length > 0?
+           <ConfirmModal
+           title={'Are you sure?'}
+           buttonConfirmName={'Delete'}
+           onClickButtonConfirm={() => {
+              deleteFunction && deleteFunction().then(()=>{
+              setRowsToBeDeleted([]);
+              setDeleteFunction(null);
+             });
+           }}
+           cancelAction={()=>{
+            setRowsToBeDeleted([]);
+            setDeleteFunction(null);
+          }}
+          disableButtons={busyDeleting}
+         >
+           
+           <p>Are you sure? You are deleting the following raster layers:</p>
+           
+           {modalDeleteContent(rowsToBeDeleted, busyDeleting)}
+           
+         </ConfirmModal>
```

Comment:
If there are many rasters selected for deletion (say 10) then the modal layout will be broken

Reply:
Fixed this in latest commit.